raft/src/main/java/org/apache/kafka/raft/internals/FuturePurgatory.java

```diff
@@ -56,7 +56,7 @@
     CompletableFuture<Long> await(T threshold, long maxWaitTimeMs);
 
     /**
-     * Complete awaiting futures whose associated values are larger than the given threshold value.
+     * Complete awaiting futures whose associated values are smaller than the given threshold value.
```

Comment:
Thanks for this fix. I think the phrase "associated values" is still misleading. How about:
> Complete awaiting futures whose threshold value from {@link await} is smaller than the given the given threshold value.